drivers/sensor/adxl367/adxl367.h

```diff
@@ -312,4 +352,13 @@ struct adxl367_dev_config {
 int adxl367_spi_init(const struct device *dev);
 int adxl367_i2c_init(const struct device *dev);
 
+#ifdef CONFIG_ADXL367_TRIGGER
+int adxl367_trigger_set(const struct device *dev,
+			const struct sensor_trigger *trig,
+			sensor_trigger_handler_t handler);
+
+int adxl367_init_interrupt(const struct device *dev);
+#endif /* CONFIG_ADT7420_TRIGGER */
```

Comment:
There's no need to guard this in the header, if you leave it in, you can make the call site much simpler.